packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -167,6 +167,7 @@ export default class InteractSubmode extends Component<Signature> {
           doc,
           relativeTo,
         );
+        await here.cardService.saveModel(here, newCard);
```

Comment:
Actually I do sus if this is the correct owner?

Reply:
it should be ok, it bascially means that the live updates for the newCard will continue to be received while the interact-submode-component is in scope.